lib/modules/keyboardNav.js

```diff
@@ -1219,8 +1219,16 @@ function reply(selected = getSelected()) {
 	));
 }
 
-function navigateTo(newWindow, href) {
-	if (newWindow) {
+function navigateTo(href, options: { newWindow?: boolean, leaveNP?: boolean }) {
+	if (options.leaveNP &&
```

Comment:
Nit: Newline before/after parentheses on multi-line conditionals

Reply:
Changed in commit #0c385ee